### PR TITLE
Add Parity swag opportunity

### PR DESCRIPTION
**Pull request by OpeOnikute**
- [x] I've checked that this isn't a new swag opportunity proposal.
- [x] I've checked that this isn't a duplicate pull request.

### CODE REVIEW

**Comment by aslafy-z** [comment]
Hello @OpeOnikute, thanks for contributing!
Can you please sort your entry in alphabetical order?

**Comment by OpeOnikute** [comment]
I've done that @aslafy-z :)

File: data.json [hunk 1]
```diff
@@ -332,6 +332,15 @@
     "dateAdded": "2018-10-20T11:38:34.000Z",
     "tags": ["clothing", "accessories", "expired", "stickers"]
   },
+  {
+    "name": "Parity Technologies",
+    "difficulty": "medium",
+    "description": "Anyone who makes any significant contribution to the Substrate ecosystem during Hacktoberfest is eligible for our own special limited edition Parity Hacktoberfest 2019 gym bag.",
```
Comment (vikaspotluri123) [inline data.json]:
```suggestion
    "description": "Anyone who makes any significant contribution to the Substrate ecosystem during Hacktoberfest is eligible for a special limited edition Parity Hacktoberfest 2019 gym bag.",
```

**Comment by vikaspotluri123** [comment]
@aslafy-z merge when you're ready 🚀

**Pull request merged**